lib/MochaWatch.js

```diff
@@ -149,15 +147,15 @@ module.exports = class MochaWatch extends EventEmitter {
 
     const file = this.sourceGraph.query({ type: "file", path });
     if (file) {
-      await file.reload();
+      file.reload();
```

Comment:
Have read through the PR a few times and this is my only query - seems that reload() is an async operation (it returns the promise from `file.load()`).

I'm not totally sure, whether it's a problem - my guess is that we could end up in a situation where we end up in a test run and trying to read relations before the load completes. I think in this case we need to trigger queue'ing in a `.then()` block attached to `reload()` to close the race.

Reply:
You're right, that should be dot-thenned :-)